scrappers/tbib/lib/Tbib.rb

```diff
@@ -0,0 +1,76 @@
+class Tbib
+	include RethinkDB::Shortcuts
+
+	def initialize()
+		@conn = r.connect(:host => $config.server_ip, :port => $config.server_port)
+		$log.log 'connected to database!'
+		@conn.use($config.database)
```

Comment:
You should check if the database exists and create it if not, there's a chance it doesn't exist just like the table

Reply:
```
if !r.db_list().run(@conn).include?($config.database)
      r.db_create($config.database)
end
```
:D

Reply:
(and with a `.run(@conn)` )